workflow/template-build-and-run-hpxml-with-stochastic-occupancy.osw

```diff
@@ -6,11 +6,25 @@
   "steps": [
     {
       "arguments": {
+        "floor_type": "WoodFrame",
+        "wall_type": "WoodStud",
+        "heat_pump_type": "none",
+        "hot_water_distribution_system_type": "Standard",
+        "lighting_present": true,
+        "clothes_washer_present": true,
+        "clothes_dryer_present": true,
+        "clothes_dryer_fuel_type": "natural gas",
+        "dishwasher_present": true,
+        "refrigerator_present": true,
+        "cooking_range_oven_present": true,
+        "cooking_range_oven_fuel_type": "natural gas",
+        "ceiling_fan_present": true,
+        "misc_plug_loads_television_present": true,
         "air_leakage_house_pressure": 50,
         "air_leakage_units": "ACH",
         "air_leakage_value": 3,
         "ceiling_assembly_r": 39.3,
-        "cooling_system_cooling_capacity": 24000.0,
+        "cooling_system_cooling_capacity": "24000.0",
```

Comment:
Ugh, really? This might be a deal breaker. This is totally confusing and no one will be able to keep track of when they need a number vs a string. I didn't realize OpenStudio was going to require this.

Reply:
Maybe we can update OpenStudio so that it allows this?